diff -up wesnoth-1.18.4/src/CMakeLists.txt.omv~ wesnoth-1.18.4/src/CMakeLists.txt
--- wesnoth-1.18.4/src/CMakeLists.txt.omv~	2025-04-11 23:47:41.009771745 +0200
+++ wesnoth-1.18.4/src/CMakeLists.txt	2025-04-11 23:50:16.742967540 +0200
@@ -253,6 +253,7 @@ if(ENABLE_GAME)
 		Boost::coroutine
 		Boost::locale
 		Boost::filesystem
+		boost_process
 		Fontconfig::Fontconfig
 		SDL2::SDL2
 		SDL2::SDL2main
@@ -349,6 +350,7 @@ if(ENABLE_SERVER)
 		Boost::coroutine
 		Boost::locale
 		Boost::filesystem
+		boost_process
 	)
 	if(MSVC)
 		target_link_options(wesnothd PRIVATE /WX)
diff -up wesnoth-1.18.4/src/filesystem.cpp.omv~ wesnoth-1.18.4/src/filesystem.cpp
--- wesnoth-1.18.4/src/filesystem.cpp.omv~	2025-04-11 23:33:17.148726685 +0200
+++ wesnoth-1.18.4/src/filesystem.cpp	2025-04-11 23:43:16.485651799 +0200
@@ -35,6 +35,8 @@
 #include <boost/iostreams/device/file_descriptor.hpp>
 #include <boost/iostreams/stream.hpp>
 #include <boost/process.hpp>
+#include <boost/process/v1/search_path.hpp>
+#include <boost/algorithm/string/replace.hpp>
 #include "game_config_view.hpp"
 
 #ifdef _WIN32
@@ -1022,14 +1024,14 @@ std::string get_exe_dir()
 	// with version
 	std::string version = std::to_string(game_config::wesnoth_version.major_version()) + "." + std::to_string(game_config::wesnoth_version.minor_version());
 	std::string exe = filesystem::get_program_invocation("wesnoth-"+version);
-	bfs::path search = bp::search_path(exe).string();
+	bfs::path search = bp::v1::search_path(exe).string();
 	if(!search.string().empty()) {
 		return search.parent_path().string();
 	}
 
 	// versionless
 	exe = filesystem::get_program_invocation("wesnoth");
-	search = bp::search_path(exe).string();
+	search = bp::v1::search_path(exe).string();
 	if(!search.string().empty()) {
 		return search.parent_path().string();
 	}
@@ -1733,7 +1735,7 @@ std::string sanitize_path(const std::str
 
 	std::string canonicalized = filesystem::normalize_path(path, true, false);
 	if(user_name != nullptr) {
-		boost::replace_all(canonicalized, user_name, "USER");
+		boost::algorithm::replace_all(canonicalized, user_name, "USER");
 	}
 
 	return canonicalized;
diff -up wesnoth-1.18.4/src/game_launcher.cpp.omv~ wesnoth-1.18.4/src/game_launcher.cpp
--- wesnoth-1.18.4/src/game_launcher.cpp.omv~	2025-04-11 23:37:39.436921443 +0200
+++ wesnoth-1.18.4/src/game_launcher.cpp	2025-04-11 23:39:29.859414237 +0200
@@ -62,6 +62,7 @@
 #include <boost/process/windows.hpp>
 #endif
 #include <boost/process.hpp>
+#include <boost/process/v1/child.hpp>
 #include <cstdlib>   // for system
 #include <new>
 #include <utility> // for pair
@@ -841,16 +842,16 @@ void game_launcher::start_wesnothd()
 	try
 	{
 #ifndef _WIN32
-		bp::child c(wesnothd_program, "-c", config);
+		bp::v1::child c(wesnothd_program, "-c", config);
 #else
-		bp::child c(wesnothd_program, "-c", config, bp::windows::create_no_window);
+		bp::v1::child c(wesnothd_program, "-c", config, bp::windows::create_no_window);
 #endif
 		c.detach();
 		// Give server a moment to start up
 		SDL_Delay(50);
 		return;
 	}
-	catch(const bp::process_error& e)
+	catch(const bp::v1::process_error& e)
 	{
 		preferences::set_mp_server_program_name("");
 
