diff -uraN wesnoth-1.16.9/src/deprecation.hpp omv-wesnoth-1.16.9/src/deprecation.hpp
--- wesnoth-1.16.9/src/deprecation.hpp	2023-04-16 03:16:27.000000000 +0200
+++ omv-wesnoth-1.16.9/src/deprecation.hpp	2023-05-03 15:53:44.192193603 +0200
@@ -14,6 +14,7 @@
 
 #pragma once
 
+#include <cstdint>
 #include <string>
 
 /** See https://wiki.wesnoth.org/CompatibilityStandards for more info. */
diff -uraN wesnoth-1.16.9/src/serialization/base64.hpp omv-wesnoth-1.16.9/src/serialization/base64.hpp
--- wesnoth-1.16.9/src/serialization/base64.hpp	2023-04-16 03:16:27.000000000 +0200
+++ omv-wesnoth-1.16.9/src/serialization/base64.hpp	2023-05-03 14:31:54.975390966 +0200
@@ -15,6 +15,7 @@
 
 #pragma once
 
+#include <cstdint>
 #include <string_view>
 #include <vector>
 
diff -uraN wesnoth-1.16.9/src/serialization/preprocessor.hpp omv-wesnoth-1.16.9/src/serialization/preprocessor.hpp
--- wesnoth-1.16.9/src/serialization/preprocessor.hpp	2023-04-16 03:16:27.000000000 +0200
+++ omv-wesnoth-1.16.9/src/serialization/preprocessor.hpp	2023-05-03 14:56:42.865972985 +0200
@@ -21,6 +21,7 @@
 #include "filesystem.hpp"
 #include "game_version.hpp"
 #include <optional>
+#include <cstdint>
 
 #include <iosfwd>
 #include <map>
